docs/src/utils/makeData.tsx

```diff
@@ -19,12 +20,12 @@ function range(len: number): Array<number> {
     }
     return arr;
 }
-const actionLinks: Array<ActionLinkItem> = [
-    {
-        label: "Edit",
-        onClick: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>, selectedRow: TableRow) => {},
-    },
-];
+// const actionLinks: Array<ActionLinkItem> = [
+//     {
+//         label: "Edit",
+//         onClick: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>, selectedRow: TableRow) => {},
+//     },
+// ];
```

Comment:
If you don't need this block of code you can just remove it instead of commenting it

Reply:
removed